### PR TITLE
Add 5.2.3 version history to docs

### PR DESCRIPTION
**Pull request by hflynn**
See https://trello.com/c/8jQ8JPgo/21-version-history-prs
Will be staged at https://www.openmicroscopy.org/site/support/bio-formats5.2-staging/about/whats-new.html when built

### CODE REVIEW

File: docs/sphinx/about/whats-new.txt [hunk 1]
```diff
@@ -1,6 +1,42 @@
...
+* CZI
+   - fixed imageCount for RGB images
+* ICS writing
+   - fixed ordering of image dimensions
+* DeltaVision
+   - fixed reading of large time dimensions
+
+Command-line tools improvements:
+
+* bftools.zip now includes the version history as :file:`NEWS.rst` (thanks to
```
Comment (mtbc) [inline docs/sphinx/about/whats-new.txt]: Ought `bftools.zip` also get the `:file:` markup?

**Comment by rleigh-codelibre** [comment]
All looks fine to me bar @mtbc's comment about the file markup.

**Comment by hflynn** [comment]
Thanks Both, I'll push another commit to fix that as soon as I finish the OMERO release tasks.

**Comment by hflynn** [comment]
Should be sorted now

**Comment by rleigh-codelibre** [comment]
Renders just fine, thanks.  Good to merge.

**Pull request merged**